CMakeLists.txt

```diff
@@ -269,7 +269,7 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
         cutlass
         GIT_REPOSITORY https://github.com/nvidia/cutlass.git
         # Please keep this in sync with CUTLASS_REVISION line above.
-        GIT_TAG v3.8.0
+        GIT_TAG 81a43e6d92cdd8c20d22392f9579604ed5f710a1 
```

Comment:
You need to set `GIT_SHALLOW FALSE` below if this is a commit

Reply:
Can this be `v3.9.0` now that it's been released?

And could you update line 254 in this file as well to keep them in sync? (I swear there was a PR that factored these out into a single variable...)